documentation/modules/auxiliary/admin/sap/sap_igs_xxe.md

```diff
@@ -0,0 +1,171 @@
+## Vulnerable Application
+
+Original SAP Internet Graphics Server (IGS) [security research][1]
+
+This module implements the SAP IGS XXE attack.
+An unauthenticated attacker can remotely read files in the server's file system, for example: /etc/passwd
+
+The Internet Graphics Service (IGS) constitutes the infrastructure to enable the application developers to display graphics
+in an Internet browser with a minimum of effort.
+The IGS has been integrated in the different SAP UI technologies from HTML GUI to Web Dynpro ABAP/Java and
+provides a server architecture where data from an SAP system or another source can be used to generate graphical or
+non-graphical output.
+
+SAP IGS versions: 7.20, 7.20EXT, 7.45, 7.49, 7.53 are affected by this vulnerability.
+Installing and Updating the IGS [instructions][2].
+Configuring the IGS [instructions][3].
+Administering the IGS [instructions][4].
+
+Once set up and configured, the instances will be vulnerable on the default HTTP port 40080.
+
+## Verification Steps
+
+  1. Install the module as usual
+  2. Start msfconsole
+  3. Do: `workspace [WORKSPACE]`
+  4. Do: `use auxiliary/admin/sap/sap_igs_xxe`
+  5. Do: `set RHOSTS [IP]`
+  6. Do: `set FILE [remote file name]`
+  7. Do: `set SHOW [true|false]`
+  8. Do: `set action READ`
+  9. Do: `check`
+  10. Do: `run`
+
+## Options
+
+### FILE
+
+File to read from the remote server, example: `/etc/passwd`
+
+### SHOW
+
+`true` - show remote file content and save in workspace loot
+
+`false` - not show remote file content and save in workspace loot
+
```

Comment:
This option is never used in your module's code and as far as I can tell is extra code that is never used. Please either implement this functionality or remove this option from your code and the associated documentation.

Reply:
This will be resolved in the next commit, feel free to reopen this if you add in the functionality.